app.config.yaml

```diff
@@ -19,10 +19,36 @@ application:
             function: actions/pdp-renderer/index.js
             web: 'yes'
             runtime: nodejs:22
+            inputs:
+              OVERLAY_PATHS:
+                - /en-us/products
+              TEMPLATES:
```

Comment:
what's the purpose of these template names? if you need an array of all templates, I would just put a constant array somewhere in the code that passing as action params here
action params are normally for variables from env file